### ISSUE TITLE
Completed Angebot kann nicht approved werden.

### ISSUE THREAD

**Issue by Twiek** [clarat-org/clarat#1428]
Angebot mit der ID 26363.

Man kriegt nur die Möglichkeit 'return_to_editing'.

**Comment by KonstantinKo** [comment]
Ja, diese Meldung bekommen wir ab und zu mal, meistens direkt von researchern. Bisher in 100% der Fällen wurde nicht beachtet, dass Angebote erst approved werden können, wenn alle Organisationen dieses Angebots approved wurden. Wissen die Researcher eigentlich, wird aber ab und zu übersehen. Ist also kein Bug.

**Comment by mokus80** [comment]
https://github.com/clarat-org/claradmin/pull/207/files
